server/controllers/all.js

```diff
@@ -13,6 +15,9 @@ import { makeLogger, KError, asyncErr } from '../helpers';
 let log = makeLogger('controllers/all');
 
 const ERR_MSG_LOADING_ALL = 'Error when loading all Kresus data';
+const ENCRYPTION_ALGORITHM = 'aes-256-ctr';
+const PASSPHRASE_VALIDATION_REGEXP = /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}$/;
```

Comment:
If I understand correctly, this means: at least 8 chars, 1 digit, 1 lowercase letter, 1 uppercase letter, correct?
I wish we could do something simpler, like just asking for 16 chars or so...